sdk/cosmos/azure_data_cosmos/examples/cosmos/patch.rs

```diff
@@ -48,7 +48,7 @@ impl PatchCommand {
         match response {
             Err(e) if e.http_status() == Some(StatusCode::NotFound) => println!("Item not found!"),
             Ok(r) => {
-                let item: serde_json::Value = r.deserialize_body_into().await?;
+                let item: serde_json::Value = r.into_json_body().await?;
```

Comment:
I don't love the naming either, but I'll open a tracking issue because I don't want to get totally bogged down in (fully legitimate, but less time-sensitive) naming bikesheds.

The main value of `into_json_body` is that it aligns with `into_raw_body` and `into_body`. A clearer name might be `into_model_from_json` or just `into_json_model`, but I'll open an issue to have that debate.

Reply:
https://github.com/Azure/azure-sdk-for-rust/issues/1957